testdata/certs/dev-ca.crt

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Are we checking these in? There was mention of creating them on each run (perhaps in SetupSuite()?

Reply:
Yeah I've raised this as a new ticket. It's next on the agenda